pkg/DispatchFunctionScaling/DispatchFunctionScaling_suite_test.go

```diff
@@ -0,0 +1,127 @@
+package DispatchFunctionScaling_test
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os/exec"
+	"sort"
+	"strconv"
+	"sync"
+	"testing"
+	"time"
+
+	. "github.com/nickaashoek/benchmark/pkg/DispatchFunctionTiming"
+	. "github.com/nickaashoek/benchmark/pkg/dispatch-reporter"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/types"
+	. "github.com/onsi/gomega"
+	"gonum.org/v1/plot"
+	"gonum.org/v1/plot/plotter"
+	"gonum.org/v1/plot/plotutil"
+	"gonum.org/v1/plot/vg"
+)
+
+var (
+	wg          sync.WaitGroup
+	testFunc    string
+	outputFile  string
+	runFunction *exec.Cmd
+)
+
+func MkPlot(measurements map[string]types.SpecMeasurement) {
```

Comment:
Plotter seems like an isolated functionality that deserves separate package/file